paparazzi/src/test/java/app/cash/paparazzi/InterceptorRegistrarTest.kt

```diff
@@ -1,4 +1,4 @@
-package app.cash.paparazzi.agent
+package app.cash.paparazzi
```

Comment:
we can still keep the package structure, i.e., move this into an `agent` subfolder/package.